dags/landsat_scenes_sync/landsat_scenes_gap_report.py

```diff
@@ -196,6 +205,59 @@ def request_usgs_api(url: str):
                 yield future.result()
 
 
+def check_for_stac_files(dest_paths):
+    """
+    If there is a missing stac, either ST or SR, and the stac is present in USGS, consider a not transfer scene
```

Comment:
"consider a not transfer scene" - I don't know what this means.

Reply:
means that I won't add to the list of transferred scenes and the system will see that as a missing scene and add to the report, then the gap filler process will try to transfer this scene again, skipping any already transferred file.